core/flyout_horizontal.ts

```diff
@@ -267,29 +267,35 @@ export class HorizontalFlyout extends Flyout {
     for (let i = 0, item; (item = contents[i]); i++) {
       if (item.type === 'block') {
         const block = item.block;
-        const allBlocks = block!.getDescendants(false);
+        // CHANGE: Checked whether block is undefined:
+        if (typeof block === "undefined") {
```

Comment:
This will work if block is undefined, but if block is null then this won't catch it. Just to be safe, I recommend using:

if (block === undefined || block === null) {

Reply:
Hi John, could you please also grant me the access to push code? Thank you very much!